src/domains/profile/pages/MyRegistrations/MyRegistrations.tsx

```diff
@@ -74,8 +74,20 @@ export const MyRegistrations = ({ blockchainRegistrations, businessRegistrations
 	};
 
 	useEffect(() => {
-		const delegateRegistrations = activeProfile.registrationAggregate().delegates();
-		setDelegates(delegateRegistrations);
+		const fetchBusinesses = async () => {
+			setIsLoading(true);
+
+			activeProfile.entityRegistrationAggregate().flush();
+
+			const businessRegistrations = await activeProfile.entityRegistrationAggregate().businesses();
```

Comment:
@faustbrian `entityRegistrationsAggregate()` is not returning, nor throwing an error for an empty profile to continue with updating the state below and is stuck in loading. Any ideas?

Reply:
What should it be throwing? If there are no wallets it won't do anything and return an empty `ExtendedTransactionDataCollection`.

Reply:
The problem is that it is not returning and below code is not reached.

Reply:
What test exactly is having issues with this? Based on logging it returns just fine.

Reply:
Unit test in "should render empty state" fails. The empty state is never rendered for an empty profile (with no registrations). Let me have a second look

Reply:
Fix coming in 0.9.333

Reply:
Cool! as I saw it happened here. No exception when syncedWallets has 0 length. https://github.com/ArkEcosystem/platform-sdk/blob/17752b5acf07bc25bf21b569aa7121f5c2448764/packages/platform-sdk-profiles/src/profiles/aggregates/entity-aggregate.ts#L47

An empty response of ExtendedTransactionDataCollection would be better instead of throwing right?